### DISCUSSION TITLE
How to add repos?

### DISCUSSION THREAD

**Discussion by gravelld** [szyb/synopackage_dotnet#259] [General]
Hi, I have created package repos at:

https://www.blisshq.com/repo/syno/packages-DSM7.json

https://www.blisshq.com/repo/syno/packages-pre-DSM7.json

Is it possible to add this to your search list?

**Comment by szyb** [comment]
Hi @gravelld !

Yes, I will check them (if they work well) and add them. 
Thanks for sharing!

**Reply by szyb** [reply]
Now they should be online.
Enjoy!

**Comment by gravelld** [comment]
@szyb Awesome, thanks! Great work on the repo search in general!

**Reply by szyb** [reply]
Thanks :)
